modules/NetworkSplitter3D/src/main/java/org/alexandrebarao/networksplitter3d/NetworkSplitter3D.java

```diff
@@ -0,0 +1,502 @@
+                    this, Integer.class,
+                    "Z-Maximum Level",
+                    NETWORKSPLITTER3D,
+                    "Z-maximum levels to compute node clusters. Node column '[z]' is needed in Gephi Data Laboratory. Note: you can use the RESET button to layout your original network.",
+                    "getZlevels", "setZlevels"));
+               
+             properties.add(LayoutProperty.createProperty(
+                    this, Integer.class,
+                    "Z-Distance Factor",
+                    NETWORKSPLITTER3D,
+                    "Layer distance between each Z-Level layout. Used to multiply Z-Scale. Note: you can use the RESET button to layout your original network.",
+                    "getZdistance", "setZdistance"));
+            
+             properties.add(LayoutProperty.createProperty(
+                    this, Float.class,
+                    "Z-Scale",
+                    NETWORKSPLITTER3D,
+                    "The vertical Z-Scale (pixels).  Note: you can use the RESET button to layout your original network.",
+                    "getVerticalScale", "setVerticalScale"));
+                     
+              properties.add(LayoutProperty.createProperty(
+                    this, Double.class,
+                    "Alfa",
+                    NETWORKSPLITTER3D,
+                    "The rotation angle (degrees from 0 to 360). Note: you can use the RESET button to layout your original network.",
+                    "getAlfa", "setAlfa"));
+              
+                         
+            
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        return properties.toArray(new LayoutProperty[0]);
+    }
+
+     public double getDataLaboratoryValue(Node n, Column col) {
+        
+                   double z = 0f;
+                   
+                   
+                   if ( col.getTypeClass()== String.class ) { 
+                        String colString = (String) (n.getAttribute(col));
+                        z = Double.valueOf(colString).doubleValue();
+                   }
+                   
+                   if ( col.getTypeClass()== Integer.class ) { 
+                        Integer colString = (Integer) (n.getAttribute(col));
+                        z = colString.intValue();
+                   }
+                   
+                   if ( col.getTypeClass()== Float.class ) { 
+                        Float colString = (Float) (n.getAttribute(col));
+                        z = colString.floatValue();
+                   }
+                   
+                   if ( col.getTypeClass()== Double.class ) { 
+                        Double colString = (Double) (n.getAttribute(col));
+                        z = colString.doubleValue();
+                   }
+                   
+                   return z;
+    }
+    
+     
+      public Node searchNode(Graph graph, int id) {

```

Comment:
You probably can replace all this code with https://gephi.org/gephi/0.9.1/apidocs/org/gephi/graph/api/Graph.html#getNode-java.lang.Object-